dbt_meshify/dbt_projects.py

```diff
@@ -19,17 +20,33 @@ class LocalDbtProject:
     def __init__(self, relative_path_to_project: str) -> None:
         self.relative_path_to_project = relative_path_to_project
         self.manifest = self.get_project_manifest()
+        self.subprojects = []
     
     def path_to_project(self) -> str:
         return os.getcwd() + '/' + self.relative_path_to_project
-    
-    def get_project_manifest(self) -> dict:
+
+    def dbt_operation(self, operation: List[str]) -> dbtRunnerResult:
         start_wd = os.getcwd()
         os.chdir(self.path_to_project())
-        result = dbt_runner.invoke(["--log-level", "none", "parse"])
+        result = dbt_runner.invoke(operation)
         os.chdir(start_wd)
-        if result.success:
-            return result.result
+        return result
```

Comment:
I love that you added this method for interacting with the dbtRunner. This makes it _waaaaaaay_ easier to keep track of directories and paths.